src/interventions/to-json.js

```diff
@@ -0,0 +1,46 @@
+/**
+Copyright 2016 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * README
+ *
+ * This intervention adds toJSON to the PaymentResponse prototype on Chrome 54.
+ */
+
+const toJSONShim = function() {
+  const instrumentObject = {
+    methodName: this.methodName,
+    details: this.details,
+    shippingAddress: this.shippingAddress || null,
```

Comment:
Since `shippingAddress` is not a dictionary you need something like this:

```
var toDict = function(addr) {
  var dict = {};
  if (addr) {
    dict.country           = addr.country;
    dict.region            = addr.region;
    dict.city              = addr.city;
    dict.dependentLocality = addr.dependentLocality;
    dict.addressLine       = addr.addressLine;
    dict.postalCode        = addr.postalCode;
    dict.sortingCode       = addr.sortingCode;
    dict.languageCode      = addr.languageCode;
    dict.organization      = addr.organization;
    dict.recipient         = addr.recipient;
    dict.phone             = addr.phone;
  }
  return dict;
};
```